chatAppTemplate_2/chatApp.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html >
+  <head>
+    <meta charset="UTF-8">
+    <title>Superb Owl</title>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    	<div class="ui">
+		<div class="left-menu">
+				<form action="#" class="search">
+					<input placeholder="search..." type="search" name="" id="">
+					<input type="submit" value="&#xf002;">
+				</form>
+				<menu class="list-friends">
+					<li>
+						<img width="50" height="50" src="http://cs625730.vk.me/v625730358/1126a/qEjM1AnybRA.jpg">
```

Comment:
Lets go ahead and download these images and upload to our github. Create a folder and put them there. That way we're not fapping about with some weird website.